lib/diff_web/controllers/page_controller.ex

```diff
@@ -114,14 +114,17 @@ defmodule DiffWeb.PageController do
     end)
   end
 
-  defp render_diff(package, from, to, stream) do
+  defp render_diff(package, from, to, stream, conn) do
     path = tmp_path("html-#{package}-#{from}-#{to}-")
 
     File.open!(path, [:write, :raw, :binary, :write_delay], fn file ->
       Enum.each(stream, fn
         {:ok, patch} ->
           html_patch =
-            Phoenix.View.render_to_iodata(DiffWeb.RenderView, "render.html", patch: patch)
+            Phoenix.View.render_to_iodata(DiffWeb.RenderView, "render.html",
+              patch: patch,
+              conn: conn
```

Comment:
Is this left over? I don't see where the `conn` is used in the template.

Reply:
yeah, sorry i just put up a new commit.